### ISSUE TITLE
All 200 videos error since 2019-10-23-00Z

### ISSUE THREAD

**Issue by EwoutH** [thulle/yt-av1#1]
As can be seen on https://htmlpreview.github.io/?https://github.com/thulle/yt-av1/blob/master/yt-av1.html, all 200 videos error starting at 2019-10-23-00Z. Did Youtube break something?

**Comment by thulle** [comment]
Sorry, I didn't notice the issue notification until now. Thanks for reporting though!

The server was banned from youtube due to the crawling, lowering the interval to 24h instead of 6h and leaving it a couple of days fixed the issue. 

Found some posts on r/datahoarder that indicated others (also only crawling, no fetching videos) were getting banned too.

**Comment by EwoutH** [comment]
Unfortunately it's erroring again

**Comment by thulle** [comment]
Tried a few workarounds and the only thing that let me list anything at all was logging in with a google account and exporting the cookies for the script to use. I aborted it after a short while so the scheduled run won't push it over the limit if there still is one, so we'll see tonight if it works out.

Youtube-dl apparently doesn't follow request ratelimits (sleeps inbetween requests) when there's no video downloaded, so if I still trigger some anti-bot measure I'll have to export the urls from the playlist and check them separately spread out over the day.
